pandas/tests/io/generate_legacy_storage_files.py

```diff
@@ -210,26 +219,36 @@ def create_pickle_data():
         "dup": DataFrame(
             np.arange(15).reshape(5, 3).astype(np.float64), columns=["A", "B", "A"]
         ),
-        "cat_onecol": DataFrame({"A": Categorical(["foo", "bar"])}),
+        # "cat_onecol": DataFrame({"A": Categorical(["foo", "bar"])}),
+        "cat_onecol": DataFrame(
+            {
+                "A": Categorical.from_codes(
+                    [1, 0], categories=Index(["bar", "foo"], dtype="object")
+                )
+            }
+        ),
```

Comment:
@jorisvandenbossche to get the old behavior here, the code changes are a bit more involved. I've not got round to reviewing all the migration guides/release notes yet. Is this included? if not, should it be?